easybuild/easyconfigs/l/LLVM/LLVM-3.9.0-foss-2016b.eb

```diff
@@ -30,7 +30,7 @@ dependencies = [
 
 configopts = '-DBUILD_SHARED_LIBS=ON '
 # required to install extra tools in bin/
-configopts += '-DLLVM_INSTALL_UTILS=ON -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_ENABLE_ZLIB=ON '
+configopts += "-DLLVM_INSTALL_UTILS=ON -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_ENABLE_ZLIB=ON -DLLVM_ENABLE_RTTI=1 "
```

Comment:
if you change the quotes on this line, also do it on the others

Reply:
I did where I could, but in some places there's a technical reason to use outer single quotes (which I prefer not to use when I can, for values that include spaces).